### DISCUSSION TITLE
How to Uninstall or remove Fooocus from windows system?

### DISCUSSION THREAD

**Discussion by bothraman** [lllyasviel/Fooocus#3290] [Q&A] [answered]
I am using Windows 11 and having trouble to uninstall or remove Fooocus from my system, as I am trying to reinstall because of a error

**Comment by mashb1t** [comment, marked as answer]
If you did download the zip file, you can just delete the zip file and the extracted folder to completely remove Fooocus.
For other installation methods you will have to delete the python packages manually.

**Reply by bothraman** [reply]
can you help me with some minor error while installing this?

**Comment by koruptioner** [comment]
[Feature Request] Straight-line mask with Shift+click in Inpaint
Requested by multiple users (including non-technical ones).
Problem:
Currently, the Inpaint mask tool only supports freehand brush strokes. Users who need precise straight edges (e.g., walls, furniture, UI elements) struggle with shaky lines.
Solution:
Add a simple Shift+click behavior:
Hold Shift.
Click point A, then click point B.
Draw a straight line between the two points (similar to Photoshop or Paint).
This small addition would save time and improve precision for many users.
Thank you for considering this!
